lib/vector/Vlib/build_nat.c

```diff
@@ -16,6 +16,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 #include <sys/types.h>
+#include <inttypes.h>
```

Comment:
inttypes.h is only available with the C standard c99. GRASS is currently supporting c89 with extensions as in gnu89. Back then, Glynn Clements had a strong opinion against switching to c99 because different compilers did not fully support c99.  This is some years ago, maybe all current compilers fully support c99, or at least inttypes.h. Apparently the github tests are passed, but I would appreciate a statement to make c99 a requirement on the GRASS dev ml.

Reply:
I put up the question on dev-list. See also comment below.